.github/workflows/index-autocomplete-search.yml

```
@@ -0,0 +1,57 @@
+
+jobs:
+  index-autocomplete-elasticsearch:
+    if: ${{ github.repository == 'github/docs-internal' }}
+    runs-on: ubuntu-20.04-xl
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - uses: ./.github/actions/node-npm-setup
+
+      - uses: ./.github/actions/setup-elasticsearch
```

actionlint (1.7.4)

missing input "token" which is required by action "Set up local Elasticsearch" defined at "./.github/actions/setup-elasticsearch". all required inputs are "token"

(action)